acceptance/Rakefile

```diff
@@ -28,7 +40,7 @@ module HarnessOptions
     end
 
     def get_options(file_path)
-      puts "Attempting to merge config file: #{file_path}"
+      puts file_path
```

Comment:
there are a few things here that are worth preserving that we can do later. they should all be captured here: https://github.com/puppetlabs/puppet/pull/4539